terraform.tfstate

```diff
@@ -0,0 +1,151 @@
+{
```

Comment:
The state file should also be added to .gitignore. you dont want that pushed to github.